.github/workflows/build.sh

```diff
@@ -31,11 +31,15 @@ else
     export MDBOOK_OUTPUT__HTML__REDIRECT='{}'
 
     # Include language-specific Pandoc configuration
-    [ -f "pandoc/$book_lang.yaml" ] && export MDBOOK_OUTPUT__PANDOC__PROFILE__PDF__DEFAULTS="pandoc/$book_lang.yaml"
+    if [ -f ".github/pandoc/$book_lang.yaml" ]; then
+        export MDBOOK_OUTPUT__PANDOC__PROFILE__PDF__DEFAULTS=".github/pandoc/$book_lang.yaml"
+    fi
 fi
 
 mdbook build -d "$dest_dir"
-[ -f "$dest_dir/pandoc/pdf/comprehensive-rust.pdf" ] && mv "$dest_dir/pandoc/pdf/comprehensive-rust.pdf" "$dest_dir/html/"
+if [ -f "$dest_dir/pandoc/pdf/comprehensive-rust.pdf" ]; then
+    mv "$dest_dir/pandoc/pdf/comprehensive-rust.pdf" "$dest_dir/html/"
+fi
```

Comment:
@max-heller, I wanted to ask about this but forgot: when would the PDF not exist?

When `build.sh` is run in CI, the `mdbook-pandoc` plugin should always be installed and so the PDF should always exist, no?

If that's true, then I would like to remove the conditional.

Reply:
If this is only ever called from the CI job with `mdbook-pandoc` installed then yes, it should always exist (unless the build fails, which will abort the script before reaching this check)

Reply:
Excellent, then I'll take it out for now: #1713. I have toyed with the idea of suggesting to people to use `build.sh` for local development, but for now it's only used in CI.

@sakex has been working on a new `mdbook` output format which would build all translations in one go: https://github.com/google/mdbook-i18n-helpers/pull/84. That would need to learn the logic of `build.sh` and then this could become the suggest way for people to build things locally.